run.py

```diff
@@ -84,7 +84,7 @@ def _find_submissions_for_contest(contest_path):
     for _, _, files in walk(contest_path):
         for filename in files:
             submission, ext = os.path.splitext(filename)
-            if ext in ALLOWED_EXT and filename[-8:] != "_test.go":
+            if ext in ALLOWED_EXT:
```

Comment:
We should keep this line, so that local test files do not conflict with the runner